ee/agent/reset.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/kolide/launcher/pkg/traces"
 )
 
-type dbResetRecord struct {
+type DBResetRecord struct {
```

Comment:
I'm not totally sure why this is exported

Reply:
oh I might be able to un-export this one too after moving to using `agent.GetResetRecords`, will give that a try